splice_machine/src/main/java/com/splicemachine/derby/serialization/ActivationSerializer.java

```diff
@@ -119,6 +119,7 @@ private void visitBaseActivationFields() throws IOException, IllegalAccessExcept
             try{
                 visit(baseActClass.getDeclaredField("row"),baseFields);
                 visit(baseActClass.getDeclaredField("resultSet"),baseFields);
+                visit(baseActClass.getDeclaredField("materialized"),baseFields);
```

Comment:
If this code is copying field values for a clone operation, should we perhaps handle all of the field values defined in the BaseActivation class?

Reply:
Thanks @msirek . I also had a question why we didn't put all the fields defined in the BaseActivation class in visitBaseActivationFields() in the first place, but couldn't get an answer that I'm confident. My guess is that most of the fields are no longer used in a copy or could be re-computed, and serialize the value may add CPU overhead and also increase the size of the serialized object? Since I don't fully understand the usage of all the variables. I chose to be conservative and only make changes to the one that is known to be problematic.